## RECOMB 2026 Accepted Papers

- Quartet-based species tree methods enable fast and consistent tree of blobs reconstruction under the network multispecies coalescent
*Junyan Dai, Yunheng Han and Erin Molloy*

- Fast and Flexible Flow Decompositions in General Graphs via Dominators
*Francisco Sena and Alexandru I. Tomescu*

- Predicting interaction-specific protein–protein interaction perturbations by missense variants with MutPred-PPI
*Ross Stewart, Florent Laval, Michael Calderwood, Matthew Mort, David Cooper, Marc Vidal and Predrag Radivojac*

- KuPID: Kmer-based Upstream Preprocessing of Long Reads for Isoform Discovery
*Molly Borowiak and Yun William Yu*

- Private Information Leakage from Polygenic Risk Scores
*Kirill Nikitin and Gamze Gursoy*

- Privacy-Preserving Pangenome Graphs
*Jacob Blindenbach, Shaunak Soni and Gamze Gursoy*

- ProtFlow: Flow Matching-based Protein Sequence Design with Comprehensive Protein Semantic Distribution Learning and High-quality Generation
*Zitai Kong, Yiheng Zhu, Yinlong Xu, Mingze Yin, Tingjun Hou, Jian Wu, Hongxia Xu and Chang-Yu Hsieh*

- POTTR: Identifying Recurrent Trajectories in Evolutionary and Developmental Processes using Posets
*Sara C. Schulte, Henri Schmidt, Martin Jürgens, Gunnar W. Klau, Palash Sashittal and Benjamin J. Raphael*

- SpecLig: Energy-Guided Hierarchical Model for Target-Specific 3D Ligand Design
*Peidong Zhang, Rong Han, Xiangzhe Kong, Ting Chen and Jianzhu Ma*

- STELAR-X: Scaling Coalescent-Based Species Tree Inference to 100,000 Species and Beyond
*Anik Saha and Md Shamsuzzoha Bayzid*

- A Cophylogenetic Approach for Virus-Host Interaction Prediction
*Md Zarzees Uddin Shah Chowdhury, T. M. Murali and Palash Sashittal*

- Denoising single-cell transcriptomic data with sparse PCA
*Victor Chardès*

- Prediction of lifetime disease liability from EHR features
*Yazheng Di and Na Cai*

- Quantum and Temporal Graph Neural Networks Reveal New Accuracy Limits in Predicting Protein–Ligand Dissociation Kinetics
*Azamat Salamatov and Gowtham Atluri*

- MaxGeomHash: An Algorithm for Variable-Size Random Sampling of Distinct Elements
*Mahmudur Rahman Hera, David Koslicki and Conrado Martínez*

- Evolutionary dynamics under phenotypic uncertainty
*Vaibhav Mohanty, Anna Sappington, Eugene Shakhnovich and Bonnie Berger*

- Deconvolving Phylogenetic Distance Mixtures
*Shayesteh Arasti, Ali Osman Berk Şapcı, Eleonora Rachtman, Mohammed El-Kebir and Siavash Mirarab*

- Achieving spatial multi-omics integration from unaligned serial sections with DIME
*Pengyu Sun, Tian Mou, Xubin Zheng and Xinlei Huang*

- Deep genomic models of allele-specific measurements
*Xinming Tu, Alexander Sasse, Kaitavjeet Chowdhary, Anna E. Spiro, Liang Yang, Maria Chikina, Christophe O. Benoist and Sara Mostafavi*

- On Deriving Synteny Blocks by Compacting Elements
*Leonard Bohnenkämper, Luca Parmigiani, Cedric Chauve and Jens Stoye*

- PaNDA: Efficient Optimization of Phylogenetic Diversity in Networks
*Niels Holtgrefe, Leo van Iersel, Ruben Meuwese, Yukihiro Murakami and Jannik Schestag*

- Information Geometry Reconciles Discrete and Continuous Variation in Single-Cell and Spatial Transcriptomic Analysis
*Jinpu Cai, Yuxuan Wang, Yunhao Qiao, Cheng Wang, Ziqi Rong, Luting Zhou, Haoyang Liu, Meng Jiang, Hongbin Shen, Jingyi Jessica Li and Hongyi Xin*

- Bacterial protein function prediction via multimodal deep learning
*Giulia Muzio, Michael Adamer, Leyden Fernandez, Lukas Miklautz, Karsten Borgwardt and Kemal Avican*

- Alternet: Alternative splicing-aware gene regulatory network inference
*Juliane Hoffmann, Julia Wallnig, Ziheng Dai, Olga Tsoy, David B. Blumenthal and Anne Hartebrodt*

- Bias in genome-wide association test statistics due to omitted interactions
*Burak Yelmen, Merve Nur Güler, Tõnu Kollo, Märt Möls, Guillaume Charpiat and Flora Jay*

- Joint Learning of Drug-Drug Combination and Drug-Drug Interaction via Coupled Tensor-Tensor Factorization with Side Information
*Xiaoge Zhang, Zhengyu Fang, Kaiyu Tang, Huiyuan Chen and Jing Li*

- Sequence-to-graph alignment based copy number calling using a network flow formulation
*Hugo Magalhães, Jonas Weber, Gunnar W. Klau, Tobias Marschall and Timofey Prodanov*

- Fast, accurate construction of multiple sequence alignments from protein language embeddings
*Minh Hoang, Isabel Armour-Garb and Mona Singh*

- SLAB: A Sweep Line Algorithm in PBWT for Finding Haplotype Block Cores
*Ardalan Naseri, Ahsan Sanaullah, Shaojie Zhang and Degui Zhi*

- Compressed inverted indexes for scalable sequence similarity
*Florian Ingels, Lea Vandamme, Mathilde Girard, Clément Agret, Bastien Cazaux and Antoine Limasset*

- Minimizer Density revisited: Models and Multiminimizers
*Florian Ingels, Lucas Robidou, Igor Martayan, Camille Marchet and Antoine Limasset*

- DiCoLo: Integration-free and cluster-free detection of localized differential gene co-expression in single-cell data
*Ruiqi Li, Junchen Yang, Pei-Chun Su, Ariel Jaffe, Ofir Lindenbaum and Yuval Kluger*

- bronko: ultrarapid, alignment-free detection of viral genome variation
*Ryan Doughty, Michael Tisza and Todd Treangen*

- Arborist: Prioritizing Bulk DNA Inferred Tumor Phylogenies via Low-pass Single-cell DNA Sequencing Data
*Leah Weber, Chi Yin Ching, Chrisopher Ly, Yixiao Cheng, Chunxu Gao and Peter Van Loo*

- Causal gene regulatory network inference from Perturb-seq via adaptive instrumental variable modeling
*Zhongxuan Sun, Hyunseung Kang and Sunduz Keles*

- Summarizing RNA Structural Ensembles via Maximum Agreement Secondary Structures
*Xinyu Gu, Stefan Ivanovic, Daniel*
